src/models/rule.rs

```diff
@@ -58,10 +58,10 @@ pub(crate) struct Rule {
   #[serde(default)]
   #[get = "pub"]
   holes: HashSet<String>,
-  /// Additional constraints for matching the rule
+  /// Additional filters for matching the rule
   #[serde(default)]
   #[get = "pub"]
```

Comment:
I think you forgot to use the alias trick.

Reply:
Updated accordingly